### ISSUE TITLE
RNFS.mkdir throws an exception on windows

### ISSUE THREAD

**Issue by rkTri** [itinance/react-native-fs#1236]
RNFS.mkdir throws an exception on windows

```
"react-native": "0.72.7",
"react-native-windows": "0.72.32",
"react-native-fs": "2.20.0",
```

`Error
    at Object.promiseMethodWrapper [as mkdir] (&platform=windows&dev=true&hot=true&inlineSourceMap=true:4755:36)
    at mkdir (&platform=windows&dev=true&hot=true&inlineSourceMap=true:130976:26)
    at runCallEffect (&platform=windows&dev=true&hot=true&inlineSourceMap=true:595781:23)
    at runEffect (&platform=windows&dev=true&hot=true&inlineSourceMap=true:596375:9)
    at digestEffect (&platform=windows&dev=true&hot=true&inlineSourceMap=true:596434:7)
    at next (&platform=windows&dev=true&hot=true&inlineSourceMap=true:596336:11)
    at currCb (&platform=windows&dev=true&hot=true&inlineSourceMap=true:596417:9)
    at tryCallOne (&platform=windows&dev=true&hot=true&inlineSourceMap=true:38029:14)
    at &platform=windows&dev=true&hot=true&inlineSourceMap=true:38110:17
    at &platform=windows&dev=true&hot=true&inlineSourceMap=true:39352:21`

**Comment by birdofpreyru** [comment]
[#1197 &mdash; The future of **react-native-fs**](https://github.com/itinance/react-native-fs/issues/1197)

**TL;DR.:** This repo is stale, no active work happens here. I actively maintain a fork of this library, but it lives, and it is managed through a different repo &mdash; https://github.com/birdofpreyru/react-native-fs.
